core/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreTests.java

```diff
@@ -1887,4 +1887,45 @@ public void batchingShardUpdateTaskTest() throws Exception {
         // Check that cluster state update task was called only once
         assertEquals(1, restoreListener.count());
     }
-}
+
+    @Test
+    public void snapshotNameTest() throws Exception {
```

Comment:
In this test I want to make sure that all client operations are covered with this check. Not really sure how I can achieve it on the unit test level.